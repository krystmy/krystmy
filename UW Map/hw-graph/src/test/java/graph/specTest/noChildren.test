# Test printing out children of a node with no child

# Create a graph
CreateGraph graph1

# Add nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 n4
AddNode graph1 n5

# Add edges
AddEdge graph1 n1 n2 e1
AddEdge graph1 n1 n3 e2
AddEdge graph1 n1 n4 e3
AddEdge graph1 n1 n5 e4

# Print the nodes in the graph and the children of n2 n3 n4 n5
ListNodes graph1
ListChildren graph1 n2
ListChildren graph1 n3
ListChildren graph1 n4
ListChildren graph1 n5
