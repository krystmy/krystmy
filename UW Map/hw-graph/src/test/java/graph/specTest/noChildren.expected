# Test printing out children of a node with no child

# Create a graph
created graph graph1

# Add nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
added node n4 to graph1
added node n5 to graph1

# Add edges
added edge e1 from n1 to n2 in graph1
added edge e2 from n1 to n3 in graph1
added edge e3 from n1 to n4 in graph1
added edge e4 from n1 to n5 in graph1

# Print the nodes in the graph and the children of n2 n3 n4 n5
graph1 contains: n1 n2 n3 n4 n5
the children of n2 in graph1 are:
the children of n3 in graph1 are:
the children of n4 in graph1 are:
the children of n5 in graph1 are:
